model/shared/src/main/scala/hmda/model/fi/Filing.scala

```diff
@@ -17,7 +17,7 @@ case object Completed extends FilingStatus {
   override def message: String = "completed"
 }
 case object Cancelled extends FilingStatus {
-  override def code: Int = 1
+  override def code: Int = -1
```

Comment:
In `filing-status.md`, this status's code is `4`. Let's update the documentation to reflect this change.